openedx_webhooks/templates/github_pr_cover_letter.md.j2

```diff
@@ -0,0 +1,47 @@
+{{ body }}
```

Comment:
Let's make this template a default template, and have the bot check the repo in question for a checklist template first.